### ISSUE TITLE
Using GreenGenes2 as Kraken 2 custom DB

### ISSUE THREAD

**Issue by VoronDM** [DerrickWood/kraken2#774]
Until the major update of GreenGenes DB to GreenGenes2, it can be easily used as a custom 16S database for Kraken 2. This will be implemented as a simple bash script (https://github.com/DerrickWood/kraken2/blob/master/scripts/16S_gg_installation.sh). Since after a major update the structure of the GG2 database has changed, it is now not clear how it can be used with the Kraken2.
Is it plan to make db build script update for using GG2 as Kraken2 custom DB?

**Comment by AlexandreThibodeauUdM** [comment]
I think Greengene2 wil be usefull only for specific tools as it is now in a tree form from what I can understand from it. Do not think it will be usable for Kraken2.

**Comment by ChaoLi-Microbio** [comment]
I agree! GreenGenes2 should be considered as the reference database for 16s taxonomy due to its major update comparing to the older version.
